source/ember-inspector/data.md

```diff
@@ -1,40 +1,33 @@
-If you are using a data library that supports the Ember Inspector such
-as Ember Data, you can inspect your models by clicking on the `Data` menu.
+If you are using Ember Data, you can inspect your models by clicking on the `Data` menu.
```

Comment:
Thanks a thought :P I saw we still keep the reference "if you want to implement for your data lib, then check X" below.